register/templates/register/activate_email.html

```diff
@@ -0,0 +1,8 @@
+{% autoescape off %}
+Hi {{ user.username }},
+Please click on the link to confirm your registration,
+http://{{ domain }}{% url 'activate' uidb64=uid token=token %}
+
+Thank you,
+One Apply Registration Team
+{% endautoescape %}
```

Comment:
Did this file pass the black test? Why do we get the "no new line at eof" error?

Reply:
it passed both tests not sure why the error is given

Reply:
updated